Assets/Runtime/GameEvents/Game/GameEvent.cs

```diff
@@ -39,7 +40,17 @@ public void RaiseGameEvent()
                     Debug.Log($"Raise event: {name}, listener: {listener}");
                 }
 
-                listener.RaiseGameEvent();
+                try
+                {
+                    listener.RaiseGameEvent();
+                }
+                catch (Exception e)
+                {
+                    if (debug)
```

Comment:
I think "debug" here is not necessary because Debug.Log is intended to work in Debug mode.

Reply:
Hmm, I removed it, but all other `Debug.Log` are surrounded with this if, so I thought I was following a convention.

Reply:
> 
> 
> Hmm, I removed it, but all other `Debug.Log` are surrounded with this if, so I thought I was following a convention.

Its a bit confusing on how its used now. The initial idea was that those log lines would get replaced with a "stack tracer" which would allow to create an overview of how events connect with each other. Though that never came to fruition :V